src/Commands/Env/CloneContentCommand.php

```diff
@@ -60,6 +66,8 @@ public function cloneContent(
             'db-only' => false,
             'files-only' => false,
```

Comment:
Maybe balk if search-replace params added with files-only:true

Reply:
Do we need this? I don't think we need this.